src/RemoteFileAPI/MessageHandlers.ts

```diff
@@ -93,6 +116,21 @@ export const RFARequestHandler: Record<string, (message: RFAMessage) => RFAMessa
     return new RFAMessage({ result: fileList, id: msg.id });
   },
 
+  getAllFileMetadata: function (msg: RFAMessage): RFAMessage {
+    if (!isFileServer(msg.params)) return error("Message misses parameters", msg);
+
+    const server = GetServer(msg.params.server);
+    if (!server) return error("Server hostname invalid", msg);
+
+    const fileList: FileMetadata[] = [...server.scripts, ...server.textFiles].map(([filename, file]) => ({
+      filename: filename,
+      atime: file.metadata.atime,
+      mtime: file.metadata.mtime,
+      btime: file.metadata.btime,
```

Comment:
For future-proofing.
```suggestion
      ...file.metadata.plain(),
```